packages/frontend/components/Video/Video.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import ReactPlayer from 'react-player'
+import styles from './videoBoard.module.scss'
+
+export interface VideoInterface {
+  id: string
+  title: string
+  creator: string
+  url: string
+}
+
+interface VideoProps {
+  key: string
+  video: VideoInterface
+}
+
+export default function Video({ video }: VideoProps) {
+  return (
+    <div>
+      <div className={styles.cardContainer}>
+        <div className={styles.descriptor}>
+          <p className={styles.title}>{video.title}</p>
+          <p className={styles.creator}> - {video.creator}</p>
+        </div>
+        <div className={styles.playerWrapper}>
+          <ReactPlayer
+            className={styles.player}
+            url={video.url}
+            width={'100%'}
+            height={'100%'}
```

Comment:
This should be in a seperate scss file.

Reply:
well, true. There should not be any reason why it shouldnt work if i put it in the player class defined in `videoBoard.module.scss`. There shouldnt be any difference. 
However, if i do it like that some weird stuff happens, the players overlap and such.
Now this really shouldnt be happening at all and i cant find an explanation for this. I commit this change either way so you can have a look at it, weird stuff

Reply:
I'll look into this. Thank you

Reply:
fixed using the `!important` modifier

Reply:
oh well, apparently thats a bad practice

Reply:
coming from soundwave over discord:
The player component appears to require width and height to be expressed in the component params by design (it specifies its own inline css from its paramaters). I think setting the width and height to 100% in the markup and using the wrapping div to specify style in the stylesheet should not be considered bad practice here